identity/handler_test.go

```diff
@@ -673,6 +675,21 @@ func TestHandler(t *testing.T) {
 		}
 	})
 
+	t.Run("case=should fail to update identity if input json is empty or json file does not exist", func(t *testing.T) {
+                for name, ts := range map[string]*httptest.Server{"public": publicTS, "admin": adminTS} {
+                        t.Run("endpoint="+name, func(t *testing.T) {
+                                        var cr identity.AdminCreateIdentityBody
+                                        cr.SchemaID = "employee"
+                                        cr.Traits = []byte(`{"email":"` + x.NewUUID().String() + `@ory.sh", "department": "ory"}`)
+                                        res := send(t, ts, "POST", "/identities", http.StatusCreated, &cr)
+
+                                        id := res.Get("id").String()
+                                        res = send(t, ts, "PUT", "/identities/"+id, http.StatusBadRequest, nil)
```

Comment:
Here, I tried to send empty interface (also null interface) as 3rd function parameters. But those values after json encoding does not set **&b** value at line 77 as null. That's why I sent **untyped nil** value.